src/avt/VisWindow/Colleagues/VisWinRendering.C

```diff
@@ -334,6 +345,34 @@ VisWinRendering::VisWinRendering(VisWindowColleagueProxy &p) :
     factory->RegisterOverride("vtkVisItAxisActor",
                               vtkVisItViewNodeFactory::axis_act_maker);
 #endif
+
+#ifdef HAVE_ANARI
+    // Change to VERBOSITY_INFO or VERBOSITY_WARNING to see more debug messages
+    vtkLogger::SetStderrVerbosity(vtkLogger::Verbosity::VERBOSITY_ERROR);
```

Comment:
Would triggering this `vtkLogger` functionality make sense to wire this up to our `-debug` or `-dump` command-line options? I mean, maybe it gets logged to a different place than our debug logs but maybe triggering it with CL args makes sense?

Reply:
Tying this to the debug level makes sense. I'll need to figure out where to query the debug level from.

Reply:
Ok, no worries if not easily doable though either.

Reply:
I tied this to VisIt's debug levels using `DebugStream::LevelX()`

Reply:
Does the vtkLogger output get written to VisIt's debug streams?

Reply:
It writes to `std::err`. Basically the same place vtkLogger output has been going. This just ties in the verbosity of any VTK/ANARI output to the debug level set by VisIt. Does VisIt redirect `std:err` to the debug streams?

Reply:
No, std::err isn't redirected to the debug streams.

vtkDebugMacro output is captured to the streams (`Debug` has to be turned on for a given vtk object). 
I don't think we've ever utilized vtkLogger specifically in VisIt before. 

Thank you for the information. Looks like the use of vtkLogger is actually only happening within the internal vtk anari classes?

Reply:
The VTK ANARI classes use `vtkDebugMacro` and `vtkWarningMacro` too. Only the back-ends loaded at runtime use the vtkLogger in their status callbacks.

Reply:
The VTK ANARI classes use `vtkDebugMacro` and `vtkWarningMacro` too. Only the back-ends loaded at runtime use the vtkLogger in their status callbacks.

Reply:
Thanks for doing that.